InvokePlaster.ps1

```diff
@@ -281,6 +295,33 @@ __________.__                   __
                                 $prompt += " ($default)"
                             }
                         }
+                        # Prompt the user for text input.
+                        $value = PromptForInput $prompt $default
+                        $valueToStore = $value
+                    }
+                    'author' {
+                        # If no default, try to get a name from git config
+                        if ([String]::IsNullOrWhitespace($default)) {
```

Comment:
To stick with the way this script has been making this check, can we use simply `if ($default) {`.  Also, for author and email, it should allow the template author to specify a store attribute value of `encrypted`.  In this case, both of these should get the `$obscuredDefault` value and display that in the prompt.